lib/logger.js

```diff
@@ -1,27 +1,46 @@
 'use strict';
 
 const Winston = require('winston');
+const morgan = require('morgan');
+
+function Logger(level, filename) {
+  const logLevel = level || 'info';
 
-function Logger(config) {
   const logger = new Winston.Logger({
-    level: config.get('log:level'),
+    level: logLevel,
     transports: [
       new Winston.transports.Console({
         colorize: true,
         timestamp: true
       })
-    ]
+    ],
+    exitOnError: false
   });
 
-  const filename = config.get('log:filename');
-
   if (filename) {
-    logger.add(Winston.transports.File, {
-      filename
-    });
+    logger.add(Winston.transports.File, {filename});
```

Comment:
I'm kind of in favor of the "log everything to STDOUT" argument, if only because it keeps the propsd code simpler. If we continue to support file logging within propsd, we'll want to fix #101 and expose log file rotation settings in the configuration.

If we log everything to STDOUT, we should allow users to turn access logging on/off in the configuration.